frontend/src/pages/modelRegistry/ModelRegistryRoutes.tsx

```diff
@@ -11,10 +11,17 @@ const ModelRegistryRoutes: React.FC = () => (
     <Route path={'/:modelRegistry?/*'} element={<ModelRegistryCoreLoader />}>
       <Route index element={<ModelRegistry />} />
       <Route path={`${process.env.MODEL_REGISTRY_NAME}`} element={<ModelRegistry />} />
-      <Route
-        path={`${process.env.MODEL_REGISTRY_NAME}/registered_models/:registeredModelId`}
-        element={<ModelVersions tab={ModelVersionsTabs.VERSIONS} empty={false} />}
-      />
+      <Route path={`${process.env.MODEL_REGISTRY_NAME}/registered_models/:registeredModelId`}>
```

Comment:
Ok, this might change with my recent changes, can be simplified

Reply:
Ok, once yours is merged I can rebase. How do you suggest we simplify? do you just mean the `path` here will be your simpler `registeredModels/:registeredModelId` and this will otherwise be the same (we want this new nesting)?